client/ts/src/hardware/task/client.ts

```diff
@@ -381,4 +384,19 @@ export class Client implements AsyncTermSearcher<string, TaskKey, Payload> {
       },
     );
   }
+
+  async openCommandObserver<A extends UnknownRecord = UnknownRecord>(): Promise<
+    CommandObservable<A>
+  > {
+    return new framer.ObservableStreamer<Command<A>>(
+      await this.frameClient.openStreamer(TASK_CMD_CHANNEL),
+      (frame) => {
+        const s = frame.get(TASK_CMD_CHANNEL);
+        if (s.length === 0) return [null, false];
+        const parse = commandZ.safeParse(s.at(-1));
+        if (!parse.success) return [null, false];
```

Comment:
Could be good to add a console warning and/or error log here. That way if this happens it's not just silently failing.